src/sinks/elasticsearch.rs

```diff
@@ -94,6 +96,35 @@ impl ElasticSearchAuth {
     }
 }
 
+#[derive(Deserialize, Serialize, Clone, Debug)]
+#[serde(deny_unknown_fields, rename_all = "snake_case")]
+pub enum BulkAction {
+    Index,
+    Create,
+}
+
+impl Default for BulkAction {
```

Comment:
Not a big deal, but I think this could also be done as:

```rust
#[derive(Derivative, Deserialize, Serialize, Clone, Debug)]
#[serde(deny_unknown_fields, rename_all = "snake_case")]
#[derivative(Default)]
pub enum BulkAction {
   #[derivative(Default)]
    Index,
    Create,
}
```

To avoid needing to implement the `Default` trait.

Reply:
Can do